src/applications/personalization/dashboard/tests/components/PrescriptionCard.unit.spec.jsx

```diff
@@ -11,8 +11,8 @@ describe('<PrescriptionCard />', () => {
       attributes: {
         prescriptionName: 'Test name',
         isTrackable: false,
-        refillDate: '2018-03-28T05:00:00.000Z',
-        refillSubmitDate: '2019-05-28T23:27:53.000Z',
+        refillDate: '2018-03-28T12:00:00.000Z',
```

Comment:
This fix is kind of a hack. But I think that when the time is noon UTC, the date will be the same no matter which timezone the date is formatted for.